charts/velero/values.yaml

```diff
@@ -119,7 +119,7 @@ configuration:
   provider:
 
   # Parameters for the `default` BackupStorageLocation. See
-  # https://velero.io/docs/v1.5/api-types/backupstoragelocation/
+  # https://velero.io/docs/main/api-types/backupstoragelocation/
```

Comment:
Can this link, and the one below, be changed to point to the 1.6 branch? I wouldn't want to direct people to main incase there are incompatible changes.